mmdet/core/anchor/point_generator.py

```diff
@@ -68,20 +68,25 @@ def num_base_priors(self):
         return [1 for _ in range(len(self.strides))]
 
     def _meshgrid(self, x, y, row_major=True):
-        xx = x.repeat(len(y))
-        yy = y.view(-1, 1).repeat(1, len(x)).view(-1)
+        yy, xx = torch.meshgrid(y, x)
         if row_major:
-            return xx, yy
+            # warning flatten would cause error in ONNX exporting
+            return xx.reshape(-1), yy.reshape(-1)
         else:
-            return yy, xx
+            return yy.reshape(-1), xx.reshape(-1)
 
-    def grid_priors(self, featmap_sizes, device='cuda', with_stride=False):
+    def grid_priors(self,
+                    featmap_sizes,
+                    dtype=torch.float32,
+                    device='cuda',
+                    with_stride=False):
```

Comment:
I suggest putting with_strides before dtype and device

Reply:
Actually, we rarely use `with_strides`, if move it before dtype and device, it will break the habits of user that calling it with  `grid_priors(featmap_sizes, bbox_preds[0].dtype,bbox_preds[0].device)`，which is correct in `anchor_generator` for a long time.